torch/_inductor/codecache.py

```diff
@@ -146,11 +146,38 @@ def is_gcc():
     return re.search(r"(gcc|g\+\+)", cpp_compiler())
 
 
+@functools.lru_cache(1)
+def valid_vec_isa():
+    # TODO: Add ARM Vec here.
+    # Tuple(isa, number of float element)
+    vec_isa_info = [("avx512", 16), ("avx2", 8)]
```

Comment:
Nit: feels more natural to use dictionary here

Reply:
Done